README.adoc

```diff
@@ -6,68 +6,30 @@ image:https://badges.frapsoft.com/typescript/code/typescript.svg?v=101[]
 
 = Cordova Showcase Application Template
 
-A template application to showcase features.
+A template application to showcase features from link:https://github.com/aerogear/aerogear-js-sdk[Aerogear Js SDK].
 
 
 == Build Instructions
 
 === Running the App
+This application can be run as a showcase using a published version of the Aerogear Js SDK, or as a development aid to working on the SDK.
 
-Note: example application will work only when cloned directly into JS-SDK.
-See: https://github.com/aerogear/aerogear-js-sdk for details.
-
-To run the application in the browser for development, run
+==== Run Showcase
+The `master` branch will always track to the latest release of the SDK.
 
+To build the showcase app, run:
 ```
 npm install
 npm run ionic:build
-npm run ionic:serve
-```
-
-=== Building the App
-
 ```
-npm run build
+You can choose to run the application in either an emulator or a browser.
 ```
+npm run ionic:serve // to run a browser
 
+npm run ionic:android // to run in an android emulator
 
-== Work with Different Backend Services
```

Comment:
Why this is gone? This is the first thing I will ask when getting this template: "How I could make this work with my organization keycloak server"